lib/command_runner.dart

```diff
@@ -77,7 +77,14 @@ class CommandRunner<T> {
   ArgParser get argParser => _argParser;
   final ArgParser _argParser;
 
-  CommandRunner(this.executableName, this.description, {int? usageLineLength})
+  /// The maximum edit distance allowed when suggesting possible intended
+  /// commands.
+  ///
+  /// Set to `0` in order to disable suggestions, defaults to `2`.
+  final int suggestedCommandsMaxEditDistance;
```

Comment:
We're in `CommandRunner` so need to say "command" twice. How about just `suggestionDistanceLimit`?